views/home.hbs

```diff
@@ -0,0 +1,20 @@
+{{>header}}
+
+<!DOCTYPE html>
```

Comment:
`home.hbs` should not have a `<head>` section, because it is inserted into the `layouts/main.hbs` template.